packages/fxa-auth-server/test/local/payments/fixtures/stripe/taxRateFr.json

```diff
@@ -0,0 +1,16 @@
+{
+      "id": "txr_1J3CjpJNcmPzuWtRwa1MitM8",
+      "object": "tax_rate",
+      "active": true,
+      "country": "FR",
+      "created": 1623903717,
+      "description": "VAT France",
+      "display_name": "VAT",
+      "inclusive": false,
+      "jurisdiction": "DE",
```

Comment:
Fascinating.

Reply:
I might not have updated all the fields when copy/pasting one of them. 😉